content/en/xtc/loadtesting/180-reports.md

```diff
@@ -48,7 +48,11 @@ Creating an intermediate report also generates an intermediate [test result]({{<
 
 After the load test has finished, you can create as many custom reports with adjusted report settings as you need. 
 
-This can be done either by going to the [_Results_ tab]({{< relref "175-results#creating-a-report-from-results" >}}), opening the context menu of any result set and selecting _Create Report_, or by clicking the **NEW** button in the _Reports_ overview. In the latter case, the report will be created from the latest available result. When determining the latest result, XTC skips pending or failed results as these typically don’t have result data attached (yet).
+This can be done on to the _Results_ tab, either by opening the context menu of any result set and selecting [_Create Report_]({{< relref "175-results#creating-a-report-from-results" >}}), or by clicking the **NEW** button on top of the _Reports_ section. In the latter case, the report will be created from the latest available result. When determining the latest result, XTC skips pending or failed results as these typically don’t have result data attached (yet).
```

Comment:
```suggestion
This can be done on the _Results_ tab, either by opening the context menu of any result set and selecting [_Create Report_]({{< relref "175-results#creating-a-report-from-results" >}}), or by clicking the **NEW** button at the top of the _Reports_ section. In the latter case, the report will be created from the latest available result. When determining the latest result, XTC skips pending or failed results as these typically don’t have result data attached (yet).
```